lib/core/src/lib/comments/comment-list/interfaces/comment-list.token.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InjectionToken } from '@angular/core';
+
+export const ADF_COMMENT_LIST_SERVICE = new InjectionToken('ADF_COMMENT_LIST_SERVICE');
```

Comment:
is there a reason using injection token here compared to traditional type/interface?

Reply:
I used the injection token as I was following the method used in the comments component. Should I replace that?